packages/angular_devkit/schematics/src/tree/interface.ts

```diff
@@ -105,6 +105,13 @@ export interface Tree {
 }
 
 
+namespace Tree {
+  export function isTree(maybeTree: object): maybeTree is Tree {
```

Comment:
This doesn't seem to be used.

Reply:
I think it should be where useful (instead of using TreeSymbol in Tree). It also communicates the types better to TypeScript. It doens't have to be part of this PR.